<!DOCTYPE HTML>


<html lang="en-US">

<!-- Mirrored from themeblossom.net/thecause/blog/ by HTTrack Website Copier/3.x [XR&CO'2010], Wed, 13 Mar 2013 20:13:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8" /><!-- /Added by HTTrack -->
<head>

<meta charset="UTF-8">
<title>Projects</title>




<!-- STYLES -->
<link href='wp-content/themes/the-cause/styles/fonts/lora.css' rel='stylesheet' type='text/css' media='screen' id='themeFont'>
<link href='wp-content/themes/the-cause/styles/fonts2/greatVibes.css' rel='stylesheet' type='text/css' media='screen'>
<link rel="stylesheet" href="wp-content/themes/the-cause/styles/grid960.css" type="text/css" media="screen"/>
<link rel="stylesheet" href="wp-content/themes/the-cause/style.css" type="text/css" media="screen" />

<link rel="alternate" type="application/rss+xml" title="The Cause - Non-Profit WordPress Theme &raquo; Feed" href="feed/index.html" />
<link rel="alternate" type="application/rss+xml" title="The Cause - Non-Profit WordPress Theme &raquo; Comments Feed" href="comments/feed/index.html" />
<link rel="alternate" type="application/rss+xml" title="The Cause - Non-Profit WordPress Theme &raquo; Home Comments Feed" href="home/feed/index.html" />
<link rel='stylesheet' id='ewd-progress-bar-css'  href='wp-content/plugins/easy-wordpress-donations/includes/css/progress-bar.css4b1d.css?ver=1.8' type='text/css' media='all' />
<link rel='stylesheet' id='ewd-front-end-css'  href='wp-content/plugins/easy-wordpress-donations/includes/css/front-end4b1d.css?ver=1.8' type='text/css' media='all' />
<link rel='stylesheet' id='rs-settings-css'  href='wp-content/plugins/revslider/rs-plugin/css/settings1bba.css?ver=3.5' type='text/css' media='all' />
<link rel='stylesheet' id='rs-captions-css'  href='wp-content/plugins/revslider/rs-plugin/css/captions1bba.css?ver=3.5' type='text/css' media='all' />
<link rel='stylesheet' id='prettyPhotoStyle-css'  href='wp-content/themes/the-cause/js/prettyPhoto/css/prettyPhoto1bba.css?ver=3.5' type='text/css' media='all' />


<script type='text/javascript' src='wp-includes/js/jquery/jquery4511.js?ver=1.8.3'></script>
<script type='text/javascript' src='wp-content/plugins/revslider/rs-plugin/js/jquery.themepunch.plugins.min1bba.js?ver=3.5'></script>
<script type='text/javascript' src='wp-content/plugins/revslider/rs-plugin/js/jquery.themepunch.revolution.min1bba.js?ver=3.5'></script>
<script type='text/javascript' src='wp-content/themes/the-cause/js/twitter1bba.js?ver=3.5'></script>
<script type='text/javascript' src='wp-content/themes/the-cause/js/prettyPhoto/js/jquery.prettyPhoto583f.js?ver=3.1.3'></script>
<script type='text/javascript' src='wp-content/themes/the-cause/js/slides.min.jqueryc5c9.js?ver=1.1.9'></script>
<script type='text/javascript' src='wp-content/themes/the-cause/js/jquery.easing.1.34e44.js?ver=1.3'></script>
<script type='text/javascript' src='wp-content/themes/the-cause/js/jquery.uniform.min4963.js?ver=1.1'></script>
<script type='text/javascript' src='wp-content/themes/the-cause/js/tbUniform4963.js?ver=1.1'></script>
<script type='text/javascript' src='wp-content/themes/the-cause/js/themeblossom5152.js?ver=1.0'></script>

<script type="text/javascript" src="wp-content/themes/the-cause/js/jquery.cookie.js"></script>

<script type="text/javascript">
/* STYLE SWITCHER */
jQuery(document).ready(function($) {
	$("#styleChanger").addClass('hidden');
	$("#styleChanger .trigger a").click(function() {
		if ($("#styleChanger").hasClass("hidden")) {
			$("#styleChanger").animate({"left": "0px"}, "slow").removeClass('hidden');
		} else {
			$("#styleChanger").animate({"left": "-362px"}, "slow").addClass('hidden');
		}
	});
});

function strpos (haystack, needle, offset) {
  // http://kevin.vanzonneveld.net
  // +   original by: Kevin van Zonneveld (http://kevin.vanzonneveld.net)
  // +   improved by: Onno Marsman
  // +   bugfixed by: Daniel Esteban
  // +   improved by: Brett Zamir (http://brett-zamir.me)
  // *     example 1: strpos('Kevin van Zonneveld', 'e', 5);
  // *     returns 1: 14
  var i = (haystack + '').indexOf(needle, (offset || 0));
  return i === -1 ? false : i;
}


</script>
<style>
td { padding: 10px;}
.mem_name { font-weight:bold; font-size:large;}
.member { padding-left:40px;}
#news p img { height:250px; width:auto;}
</style>

</head>

<body class="page page-id-62 page-template page-template-page-blog-wide-php theme-thecause">

<a id="top"></a>

	
<!-- HEADER -->
<div id="header" class="width100" style="background-image: url('wp-content/themes/the-cause/images/backgrounds/header/headerBckg.png'); background-color: #ffffff;">
    <div class="width1000">
        
        <div id="logo"><a href="#" title="CARESbd"><img src="wp-content/uploads/sites/5/2013/01/logo2.png" alt="CARESbd"></a></div>
		
    </div>
</div>
<!-- .HEADER -->

<div class="clear"></div>

<!-- CONTENT -->
<div id="contentHolder" class="width100">
    
	<!-- Navigation -->
    <div id="navigationBckg" class="green width100">
		
        <div id="navigation">
		
		<div class="width1000">
		<ul id="menu-navigation" class="navigation"><li id="menu-item-1101" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-60 current_page_item menu-item-1101"><a href="index.php">Home</a>

</li>
<li id="menu-item-1103" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1103"><a href="#">CARES</a>
<ul class="sub-menu">
	<li id="menu-item-1108" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1108"><a href="at_a_glance.html">At a Glance</a></li>
	<li id="menu-item-1110" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1110"><a href="vision_objectives.html">Vision, Mission and Objectives</a></li>
    <li id="menu-item-1110" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1110"><a href="performance_report.html">Performance Report</a></li>
	<li id="menu-item-1106" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1106"><a href="#">Function & Administration</a>
        <ul class="sub-menu lastul">
            <li id="menu-item-1108" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1108"><a href="e_c_members.html">Executive Committee Members</a></li>
            <li id="menu-item-1110" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1110"><a href="governance.html">Governance of CARES</a></li>
            <li id="menu-item-1109" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1109"><a href="#">Membership</a></li>
            <li id="menu-item-1109" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1109"><a href="memorandum.html">Memorandum of Association</a></li>
        </ul>
    </li>
</ul>

</li>

<li id="menu-item-1107" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1107"><a href="projects.html">Projects</a></li>
<!--<li id="menu-item-1124" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1124"><a href="news.html">News</a></li>-->
<li id="menu-item-1129" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1129"><a href="current_a.html">Current Affairs</a></li>
<li id="menu-item-1129" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1129"><a href="video.html">Video</a></li>
<li id="menu-item-1129" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1129 con"><a href="#">Conference</a>
<ul class="sub-menu">
	<li id="menu-item-1108" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1108"><a href="conference">Recently concluded conference</a></li>
	<li id="menu-item-1110" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1110"><a href="conference_report.html">Conference report (Draft)</a></li>
    <li id="menu-item-1110" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1110"><a href="conference_souvenir.html">Conference souvenir</a></li>
	
</ul>
</li>
<li id="menu-item-1129" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1129"><a href="contact.html">Contact Us</a></li>

</ul>        </div>
		
        </div>
		
    </div>
	<!-- .Navigation -->
        
    <!-- MAIN -->
    <div id="main" class="width100">
	
			
		<!-- Promo Image -->
		<div id="promoImage" class="width100">
						<div style="background-image: url('wp-content/uploads/sites/5/2013/01/tb_green.png');">
				<img src="wp-content/uploads/sites/5/2013/01/tb_green.png">
							</div>
					</div>
		<!-- .Promo Image -->

    
        <!-- Content -->
        <div id="content" class="default extra-green  sidebar-green width1000" >
		<div class="fullWidth">


<h2 style="text-align:center">Current Affairs</h2>

<!-- News -->
<div id="news">

	
	<div class="news wide ">
			        
	        
        <div>
        	<h3><a href="book1.html">Proceedings of international conference on biotechnology, 25-26 May, 2013</a></h3>
            <hr/>
            <h5>PROBIOTIC TECHNOLOGY FOR SUSTAINABLE DISEASE MANAGEMENT OF PRAWN AQUACULTURE IN BANGLADESH</h5>
            <p>FARZANA E. HOSSAIN, ABU MD. RAMIM, SABIKUNNAHAR, SHAFIQUR RAHMAN, M. NIAMUL NASER<sup>1</sup> AND MUHAMMAD MANJURUL KARIM* <br> <br>
            	<i> Department of Microbiology and <sup>1</sup>Department of Zoology, Faculty of Biological Sciences, University of Dhaka, Dhaka 1000, Bangladesh</i>
            </p>
            
            <p><b>ABSTRACT</b><br> <br>
              <i>Macrobrachium rosenbergii</i>, the giant fresh  water prawn is one of the most important commercially-produced crustaceans in  Bangladesh. A significant limitation to the industry is loss of productivity  owing to the emergence of a large variety of pathogenic bacteria and viruses,  and their resistance to chemotherapeutic drugs resulting in mass mortality and  consequent crop failure. Finding a public health and environment-friendly  alternative is therefore a burning question to ensure sustainability of this  industry in Bangladesh.<br>
Molecular fingerprinting of randomly selected pathogens that  were associated with huge prawn mortality was addressed for their  identification. Probiotics – the friendly bacteria with a host of benefits that  work by competitive exclusion of pathogenic bacteria can be a suitable  alternative. This study aims to test probiotic bacteria able to inhibit growth  of pathogenic bacteria that caused prawn mortality.</p>
<p>Following a disease outbreak in prawn hatcheries of Sathkhira and Khulna  regions during May 2012, samples were collected immediately from dead prawn  larvae, rearing water and fish feed in order to isolate and characterize the  pathogenic bacteria. Subsequently, sixteen of the randomly selected isolates  were subjected to amplify their ribosomal DNA to do amplified rDNA restriction  analysis (ARDRA) followed by sequencing and phylogenetic analyses of the  isolates in order to address their identification. The antibiogram of the  isolated pathogens was conducted before they were challenged with probiotic  bacteria, <i>Lactobacillus</i> spp isolated  from curd.</p>        
			<hr/>
            <br>
            <p>*<i>Author for correspondence: Professor  Muhammad Manjurul Karim</i><br>
              <i>Department of  Microbiology.University of Dhaka, Dhaka 1000</i><br>
              <i>Tel: 01715 490535,Fax:  880 2 861 5583 </i><br>
              <i>email: </i><a href="mailto:manjur@univdhaka.edu"><i>manjur@univdhaka.edu</i></a></p>
            <p>The  bacterial isolates causing prawn mortality were identified as <i>Enterobacter </i>spp., <i>Klebsiella</i> <i>pneumoniae</i>, <i>Enterococcus</i> spp. and an extremophile <i>Exiguobacterium</i> spp. Eighteen antibiotics  were tested to study the drug-resistance pattern of the isolates. Virtually all  the tested isolates (sixteen) showed moderate to resistance states to the drugs  tested, and the absolute resistance was recorded to Ceftazidime, Oxacilin,  Penicilin G and Rifampicin. In an <i>in  vitro</i> attempt to control pathogen&rsquo;s growth, the cell-free supernatant (CFS)  of <i>Lactobacillus</i> spp produced a dose-  and age-dependent reduction of bacterial count.<br>
The inhibition of bacterial growth could be attributed to  some extracellular substances released from <i>Lactobacillus </i>spp, demonstrating its potential to be an excellent probiotic candidate  against the isolates associated with prawn mortality, a possible applications  in prawn aquaculture. If successfully translated in culture ponds, this will be  an eco-friendly approach to counter bacterial infections without compromising  the prawn quality, thereby ensuring food safety in prawn industries of Bangladesh.</p>
            <p>Keywords: Probiotic bacteria, Aquaculture, Disease, Food  safety, Bangladesh<b></b></p>
            <p><b>INTRODUCTION</b><br>
The bulk of the  global aquatic production has been exclusively attributed to Asia.  Given the nutritive values and the agro-climatic conditions in the coastal  zones, Bangladesh  (mainly South-West coastal region) offers one of the most suitable rearing  environments in the world for farming different kinds of fishes and crustaceans,  and freshwater prawn is the most notable product. Shrimp Aquaculture, popularly  regarded as Silver crop of Bangladesh,  is the country&rsquo;s second largest exportable commodity and the giant freshwater  prawn commonly known as Golda (<i>Macrobrachium  rosenbergii</i>) is one of the most important commercially produced  crustaceans. Prawn culture bagged around 25% of the total fish and fisheries  exportable commodity of Bangladesh and based on the steady progress in the last  decade, Bangladesh aims to earn US$ 1 billion in the year 2014-15 exclusively  from this sector.<sup>(1)</sup> </p>
            <p>Ironically,  this giant sector has been stumbled adversely, according to a World Bank report  that estimated the global losses due to shrimp diseases are around US$ 3  billion.<sup>(2)</sup> Shrimp farmed in both salt water and freshwater are  becoming increasingly vulnerable to bacterial infection due to the ease with  which pathogens are transmitted in aquaculture.<sup>(3)</sup> Addressing those  threats caused by virus, bacteria, fungi or protozoa has become a great challenge  to sustain and scale up this trend. Regular application of protein rich feed,  high density of prawn in hatchery tank and rearing ponds rendering as a vital  factor for immense pathogen propagation. <i>Vibrio</i> species, though comprise a major part of normal flora of crustaceans, have been  reported as the predominant causative agents for numerous disease outbreaks.<sup>(4-6)</sup>  They are ubiquitous and a predominant component of prawn culture environment.<sup>(7-10)</sup>  A significant limitation to the industry is loss of productivity owing to the  emergence of a large variety of pathogenic bacteria and viruses, and their  resistance to chemotherapeutic drugs resulting in mass mortality and consequent  crop failure.<sup>(11)</sup></p>
            <p>The use of antibiotics to control  diseases was widely practiced; however its indiscriminate use led to the  emergence of numerous <i>multiple  drug-resistant</i> (MDR) bacteria, and is now a serious issue of public health  concern. Together with MDR bacteria, the presence of <i>specific pathogenic bacteria</i> (SPB) in raw and processed prawn has  also been posing a serious threat for the viability of this export sector  because of consignment rejections by importing countries on the ground of  biosecurity. In a nutshell, their presence not only damages farm productivity,  but also affects the export performance and the situation thus affects the  whole value chain. Finding a public health and  environment-friendly alternative is therefore a burning question to ensure  sustainability of this industry in Bangladesh.</p>
            <p>The use of nonpathogenic probiotic  bacteria appeared as a novel alternative as bio-control agent for disease  prevention.<sup>(12)</sup> These so-called beneficial bacteria are not  therapeutic agents but will alter directly or indirectly the composition of the  microbial community in the rearing environment and in the shrimp gut<sup>(13-14) </sup> without causing any harm to the host<sup>(15)</sup> once applied to the rearing  tanks and ponds directly or with the feed respectively.<sup>(16)</sup>  Historically, probiotic cultures have been associated with lactobacilli,  cultured from milk and dairy products.<sup>(17)</sup> The primary antimicrobial  effect exerted by lactobacilli is by the production of lactic acid and  reduction of pH. However, there are other metabolic products such as hydrogen  peroxide, diacetyls, propionic acid, acetic acid, carbon dioxide, reuterin, and  bacteriocins reported to contribute to their antimicrobial activity.<sup>(18)</sup>  Because of its broad spectrum antimicrobial effect, selected probiotic <i>Lactobacillus</i> spp. could be a useful  alternative to antibiotic to control bacterial infection in shrimp aquaculture.</p>
            <p>The  south and south-west coastal parts of Bangladesh profoundly invest on  prawn aquaculture. An outbreak of mass mortality in post-larvae (PL) of the  Golda hatcheries of Sathkhira and Khulna  region was reported in May 2012. The damage could not be controlled by the use  of medication and other available treatments, as described by the farmers. An  earlier study from this research group isolated and partially characterized a  good number of pathogenic strains from diseased and dead PLs of prawn, rearing  waters and fish feed, and studied their drug resistance pattern.<sup>(19)</sup>  Here, we aim to further characterize sixteen multi-drug resistant (MDR) isolates  from that study at the molecular level in order to have their confirmative identification.  Finally, probiotic <i>Lactobacillus</i> strains were tested to assess its efficacy for inhibition of isolated pathogens.  The finding is expected to help us realize a better preventive approach to  control prawn mortality, and thereby strengthen the aquaculture sector  economically. </p>
            <p><b>MATERIALS AND METHODS</b></p>
            <p><i>Sampling, microbial analyses and representative  isolates</i><br>
              In  order to address the <i>M. rosenbergii</i> PL mortality observed in Khulna and Sathkhira hatcheries in May 2012, a study  was conducted that isolated and identified the bacterial pathogens  presumptively in diseased PL.<sup>(19)</sup> 16 isolates (randomly selected  from 71 isolates of that study), were taken only to be used as target organisms  and were challenged with suitable probiotic bacteria<sup>(19)</sup> that is  expected to outcompete or impede the growth of the selected pathogens.</p>
            <p><b><i>Antibiotic susceptibility</i></b><b> </b></p>
            <p>The antimicrobial  susceptibility of the bacterial isolates was determined <i>in vitro</i> by using the standardized agar-disc-diffusion method.<sup>(20)</sup> Eighteen commercially available antibiotic discs (Oxoid) and Mueller-Hinton  agar (Oxoid) media were used for the antimicrobial assay. The inhibition zone  sizes were interpreted by referring to zone diameter interpretive standards  from National Committee for Clinical Laboratory Standards (NCCLS) 2000. (Data  are available online at www.liebertpub.com/fpd).
</p>
            <p><b><i>Molecular fingerprinting </i></b></p>
            <p>Genomic DNA extraction:  DNA extraction was done using a Wizard Genomic DNA Purification Kit (Promega)  according to the manufacturer&rsquo;s instructions, and stored at 4°C for further  analysis. DNA concentration was determined using the Nanodrop ND-1000 [Nanodrop  Thermo Scientific (Fisherbrand,   USA)].
</p>
            <p><b><i>Polymerase</i></b><b><i> chain  reaction</i></b><br>
Molecular  fingerprinting of the bacterial isolates was carried out using the polymerase  chain reaction and amplified ribosomal DNA restriction analysis (ARDRA)  technique. Using the genomic DNA as template,   a PCR was performed with <i>Taq</i> 2X Master Mix (New England Biolab, UK) using universal primers (forward 27F [AGAGTTTGATCMTGGCTCAG,<sup> (21)</sup>], reverse 1492R [TACCTTGTTACGACTT,<sup> (22)</sup>]  for the amplification of bacterial 16S rDNA gene in a thermal cycler (Biometra,  Germany). The amplification was carried out with an initial denaturation stage  at 940C for 5  min, followed by 35 cycles each of denaturation (94°C for 1 min 30 s), annealing  (55°C for 1 min), and extension (68°C for 1 min 30s) followed by a final  extension of 10 min at 68°C. PCR tubes were then stored at 4°C for analysis. For  the horizontal electrophoresis, the PCR products were applied on a 1% agarose  gel (w/v) using the 1 kb plus DNA ladder (Bioneer,   South Korea). </p>
            <p><b><i>Amplified Ribosomal DNA Restriction Analysis (ARDRA)</i></b><br>
Restriction  digestion of 16S rDNA (approximately 1500 bp) purified from bacterial isolates was  performed, using the <i>Rsa</i>I (New  England biolab, UK) restriction enzyme for ARDRA.<sup>(23)</sup> The digests  were visualized after agarose gel electrophoresis, using 1% agarose (w/v) gel.  DNA marker 1Kb (Bioneer, South Korea) was used to analyze  different restriction fragments. </p>
            <p><b><i>Sequencing and phylogenetic analysis</i></b></p>
<p>The 16S rDNA PCR  products of representative isolates from each ARDRA was purified with the  Wizard PCR SV Gel and PCR Clean-Up System kit (Promega, USA) and sequenced in  5&rsquo; to 3&rsquo;,  and 3&rsquo; to 5&rsquo; directions using  forwards 27F and reverse 1492R primers respectively (ABI sequence analyzer,  USA). Partial sequences were combined to full-length sequences (1300 bp–1400  bp) via the SeqMan Genome Assembler (DNAstar, USA) and were compared to the GenBank database  of the National Center for Biotechnology Information  (NCBI) (www.ncbi.nlm.nih.gov/GenBank) by means of the basic local alignment  search tool to identify close phylogenetic relatives. Multiple sequence  alignment of the retrieved reference sequences from NCBI was performed with the  ClustalW (Larkin et al., 2007) software and was exported to the Molecular  Evolutionary Genetics Analysis (MEGA) program<sup>(24)</sup> for the  construction of phylogenetic tree using the Neighbor joining algorithm and  selecting 3000 bootstrap replication.</p>
            <p><b><i>Isolation and identification of probiotic bacteria</i></b><b> </b><br>
Curd  samples were serially diluted for enrichment in alkaline peptone medium (Oxoid, UK)  and incubated at 25oC for 90 min before plating. Diluted samples  were plated onto De Man Rogosa Sharpe (MRS) medium for <i>Lactobacillus </i>isolation  and incubated at 37<sup>o</sup>C for 48-72 h. Twenty colonies with typical  characteristics namely pure white, small (2 - 3 mm diameter) with entire  margins<sup>(25)</sup> (Figure 1) were picked from each plate and transferred  to MRS broth. Under a light microscope, the strains appeared as long rods with  no spore formation, and produced Gram +ve reaction. Following biochemical  experimentation, the isolates produced carbohydrate utilization test positive,  motility negative, urease positive, catalase negative, indole negative, and  citrate utilization negative reactions, and were presumptively identified as <i>Lactobacillus</i> spp.<b><sup>(26)</sup></b></p>
            <p style="text-align:center"><img src="img/Lactobacillus.jpg"></p>
            <p style="text-align:center"><b><i>Figure  1:</i></b><i> Pure culture of  Lactobacillus spp. isolated on MRS agar plate<b></b></i></p>
            <p><b><i>Antagonism assay</i></b></p>
<p>The possible probiotic  potential or the antagonistic activity of the isolated lactobacilli against the  selected pathogens was investigated first on a solid medium,<sup>(14-15)</sup> and later on a liquid culture. Initially, the method of disc diffusion assay  was applied on Mueller-Hinton agar medium. Nutrient broth medium was used to  harvest the <i>Lactobacillus</i> isolates  for 72 hours at 37<sup>o</sup>C without shaking in an orbital shaker. Cell-free  supernatant (CFS) were collected after centrifugation of the harvested cultures  at 8,000 <i>xg</i> for 5 min, followed by  filtration with 0.45 µm pore size (Millipore). On the other hand, each of  overnight-grown pathogenic bacteria on LB broth at 37<sup>o</sup>C with shaking  at 150 rpm, to be used as target pathogens, was swabbed over Mueller-Hinton  agar plates. Blank disc was soaked with three consecutive additions of 10 µl  CFS from probiont culture, and gently impregnated on the same medium containing  the target pathogen. A disc soaked with nutrient broth medium only was used as  a control. After impregnation of the discs, the plates were initially incubated  at 4ºC for 2-3 hours before they were transferred at 37ºC for 24 hours.  Antimicrobial activity was estimated as the diameter (mm) of clear inhibitory  zone formed around the discs. Isolates exhibiting a minimum of 15 mm zone of inhibition  around the disc were considered for their further characterization as  prospective probiotic bacteria.</p>
<p>After  a preliminary screening of prospective <i>Lactobacillus</i> isolates, the candidate probionts were then subjected to a further antagonism  assay but to a liquid medium.<sup>(27)</sup> The 1, 2 and 3 days-old CFS of <i>Lactobacillus</i> were prepared from an MRS  broth culture incubating them at 30ºC without shaking in an orbital shaker  followed by centrifuging at 8,000 <i>xg</i> for 15 min, and filtering through a 0.45 µm syringe filter (Millipore). These  were designated as CFS-1, CFS-2 and CFS-3, respectively. On the other hand,  overnight culture of a pathogenic bacterium was harvested in LB broth,  centrifuged and the pellet was washed three times with physiological saline. The  cell suspension was prepared in such a way with physiological saline so that a  cell density of approximately 2 x 107<b> </b>cells/ ml could be  made. Four 50 ml flasks, designated as I, II, III and IV, were all fed with <i>pathogenic isolates</i>, 5 ml each. These  suspensions in four flasks were supplemented with 0, 1, 2 and 5 ml of <i>Lactobacillus</i> CFS, each from  different-aged cultures, together with 5, 4, 3 and 0 ml of MRS broth culture  medium, respectively, hence the total volume were adjusted at 10 ml suspension  in each flask. While flask I was left without any probiont CFS, instead  contained only the culture medium, it thereby served as the &lsquo;control&rsquo; of the  experiment. All the twelve flasks were incubated at 37ºC following addition of  CFS. The effect of CFS on the viability of pathogenic isolates was observed at  0 and 18 hours time intervals by culturing them on TCBS media by drop plate  technique followed by incubation at 37<sup>o</sup>C. </p>
<p><b>RESULTS</b></p>
<p><b><i>Antibiotic susceptibility</i></b></p>
<p>Sixteen isolates were  challenged with 18 antibiotics (Oxoid,   UK) and the  antibiotics tested for the isolates were Ampicillin, Amoxicillin with  clavulanic acid, Aztreonam, Azithromycin, Chloramphenicol, Ceftazidime,  Ciprofloxacin, Gentamycin, Cefotaxime, Erythromycin, Nitrofurantoin, Cefepime,  Imipenem, Kanamycin, Oxytetracycline, Levofloxacin, Oxacilin, Penicillin G, and  Rifampicin. A total of 9 isolates (56%) showed sensitivity to the drugs  Ciprofloxacin, Cefotaxime, Nitrofurantoin, Kanamycin and the highest  susceptibility (88%, n = 14) found against the drug Oxytetracycline. On the  other hand, the antibiotics: Ceftazidime, Oxytetracycline, Oxacilin, Penicillin  G and Rifampicin were found absolutely ineffective against all the pathogens  tested (n= 16).  A total of 12 isolates (&gt;75%)  exhibited resistance to Ampicillin, Aztreonam, Erythromycin, 74% (n= 11) showed  resistance to Amoxicillin with clavulanic acid, 65% (n=23) to Ampicillin,  Aztreonam, and Cefepime, 50% to 60% (n= 8) to Azithromycin, Cefotaxime; and 25%  to 40% (n= 4 to 6) showed resistance to rest of the drugs: Chloramphenicol,  Ciprofloxacin, Gentamycin and Nitrofurantoin (Figure 2). It was also found that  none of the drugs appeared effective in inhibiting all the pathogens. That  means none of the drugs could be prescribed as a sole drug agent that would be  successful to kill all the tested pathogens, and the organisms harbor mechanism  to counter effects of multiple antibiotics.</p>
<p><b>Molecular  Characterization</b><br>
  <b><i>ARDRA</i></b><br>
  The genomic DNAs,  purified from the sixteen bacterial isolates were subjected to a polymerase  chain reaction in order to amplify their respective 16S rDNA. These produced  amplicons of about 1400 bp (Figure 3, indicated within a box), which were used  as substrate for cleavage by a restriction enzyme, <i>Rsa</i>I to address their ARDRA pattern. The pattern produced four  different groups for all of the 16 isolates (Figure 4). The size of the  restriction fragments and the corresponding bacterial isolates respective to  each ARDRA group are summarized in Table 1.</p>
<p><b><i>16s rDNA Gene Sequence analysis</i></b></p>
<p>From each of four  ARDRA groups, isolates were randomly selected for detailed 16s rDNA gene  sequence analyses. Phylogenetic tree of 16S rDNA gene sequences of isolates and  close relative reference isolates were retrieved from database with accession  numbers (Figure 5). The tree was generated in program MEGA 5 using the  Neighbour-Joining algorithm; with the archaeae sequences serving as outgroup.  Bootstrap values (n = 3000 replicates) are shown at branch nodes and the scale  bar represents the number of changes per nucleotide position. The figure  identifies all representative isolates from ARDRA group A: 2.2, 3.1, and 7.2, as <i>Enterobacter </i>spp. Isolates 4.2 and 6  from group B is phylogenetically closely related to <i>Klebsiella</i> <i>pneumoniae</i>,  while isolate AS1T5 from group C, and isolates AS2T4  and AS2T6 from group D are phylogenetically closely  related to <i>Enterococcus</i> spp. and <i>Exiguobacterium</i> spp. respectively, of  which the later comes from an extremophile.
</p>
<p style="text-align:center"><img src="img/book1_1_2.jpg"/> </p>
<p><b><i>Figure 2</i></b><i>: <b>Antibiogram of pathogenic bacteria</b>. The  used antibiotics were Ampicillin (AMP, 10µg, Amoxicillin with clavulanic acid  (AMK), Aztreonam (ATM, 30µg), Azithromycin (AZM, 15µg), Chloramphenicol (C,  30µg), Ceftazidime (CAZ, 30µg), Ciprofloxacin (CIP, 5µg), Gentamycin (CN,  120µg), Cefotaxime (CTX, 30µg), Erythromycin (E, 15µg), Nitrofurantoin (F,  300µg), Cefepime (FEP, 30µg), Imipenem (IPM, 10µg), Kanamycin (K, 30µg),  Oxytetracycline (OT, 30µg),</i> <i>Oxacilin  1µg  (OX, 1µg), Levofloxacin (LEV,5µg), Oxacilin  1µg  (OX, 1µg), Penicilin G (P, 10µg),  Rifampicin (RD, 5µg).</i></p>
<p style="text-align:center"><img src="img/book1_1_3.jpg"/> </p>
<p style="text-align:center"><b><i>Figure  3: Agarose gel electrophoresis of 16s rDNA gene of 16 isolates. </i></b><i>The far right lane is 1 kb DNA ladder,  while the far left lane is used as negative control.</i></p>
<p style="text-align:center"><img src="img/book1_1_4.jpg"/> </p>
<p style="text-align:center"><b><i>Figure  4: ARDRA pattern analysis.</i></b> Restriction digestion  of bacterial isolates using <i>Rsa</i>I  enzyme, lane M indicates 1Kb DNA marker. </p>
<p><b>Table 1: The ARDRA pattern of the  pathogenic isolates</b></p>
<table border="1" cellspacing="0" cellpadding="0" width="523">
  <tr>
    <td width="133"><p align="center">ARDRA GROUP </p></td>
    <td width="162"><p align="center">Fragments Size<b></b></p></td>
    <td width="228"><p align="center">Isolate ID </p></td>
  </tr>
  <tr>
    <td width="133"><p align="center">Group A </p></td>
    <td width="162"><p align="center">500, 450 and 400</p></td>
    <td width="228"><p align="center">1, 2.1, 2.2, 3.1, 3.2, 5</p></td>
  </tr>
  <tr>
    <td width="133"><p align="center">Group B </p></td>
    <td width="162"><p align="center">850 and 450</p></td>
    <td width="228"><p align="center">4.2, 6, 7.1</p></td>
  </tr>
  <tr>
    <td width="133"><p align="center">Group C </p></td>
    <td width="162"><p align="center">900 and 350</p></td>
    <td width="228"><p align="center">AS1T5</p></td>
  </tr>
  <tr>
    <td width="133"><p align="center">Group D </p></td>
    <td width="162"><p align="center">550, 450 and 250 </p></td>
    <td width="228"><p align="center">AS1T3, AS2T3 ,    AS2T4, AS2T5, AS2T6</p></td>
  </tr>
</table>
<p>&nbsp;</p>
<p><b><i>Antagonism assay in solid media</i></b></p>
<p>On Mueller-Hinton agar  plates, the isolates of <i>Lactobacillus</i> spp. recovered from curd samples showed &gt;10 mm zones of inhibition.<sup>(19) </sup> Figure 6 demonstrates two such plates, swabbed with AS<sub>5</sub>XB<sub>2-1</sub>  and
  AS<sub>4</sub>T<sub>2-1</sub>, two presumptive  pathogenic strains that caused prawn mortality. They were challenged  with CFSs of six <i>Lactobacillus</i> spp,  grown for two days in LB broth, each soaked in different blank discs, and all  of them produced clear zones of inhibition against the growth of the pathogens.  Of them, eighteen strains were found producing &gt;15 mm zones of inhibition,  and therefore were short-listed for further characterization. However, the  extent of inhibition varied depending on target pathogens.</p>
<p style="text-align:center"><img src="img/book1_1_5.png"/> </p>
<p><b><i>Figure 5:</i></b><i> <b>Phylogenetic tree with the 16S rDNA gene using the MEGA v5 program by neighbor-joining (NJ) method.</b> The dendograms based on analysis of restriction patterns of 16S rDNA obtained with Rsa</i>I<i> was built using the SeqMan Genome Assembler (Partial sequence), ClustalW (Multiple sequence), Molecular Evolutionary Genetics Analysis (MEGA)5 program and Neighbour-Joining algorithm, with the archaeae sequences serving as outgroup. Sequences of the isolates obtained in this study are shown in bold. Corresponding randomly amplified polymorphic DNA types are mentioned for the isolates.</i></p>

<p style="text-align:center"><img src="img/book1_1_6.png"/> </p>
<p><i><b>Figure 6</b>: Two representative plates, swabbed with pathogenic strains, </i><i>AS<sub>5</sub>XB<sub>2-1</sub> and </i><i>AS<sub>4</sub>T<sub>2-1</sub> </i><i>respectively were challenged with CFS of</i><i>Lactobacillus spp. each soaked in separate disc on Mueller-Hinton media. Zones of inhibition (marked by red arrow) were formed around the discs after a period of 24-hour incubation at 37<sup>o</sup>C.</i></p>
<p><b><i>ANTAGONISM ASSAY IN LIQUID MEDIA</i></b></p>
<p>In order to address the probiotic potential of one of the isolated <i>Lactobacillus spp., </i>PLB<sub>17</sub> against the pathogen suspended in a liquid state, effect of cell-free supernatant (CFS) of the <i>Lactobacillus spp.</i> on washed cells of AS<sub>1</sub>T<sub>5</sub>, an isolated pathogenic species<i>,</i> in sterile saline suspension was investigated. Four flasks, each containing a unified concentration of the pathogenic bacteria spp. was mixed with different amount of CFS of <i>Lactobacillus</i> spp. and the mixtures were incubated at 37<sup>o</sup>C for 18 hours. An initial population of 107 cells/ ml ended up to complete elimination of the bacterial count upon addition of CFS in different volumes only within 18 h of incubation, while no significant changes of growth was observed in the flask which only contained the medium, but no CFS (Figure 7, <i>control</i> part, or box 1), indicating the growth inhibition of the pathogen was brought forward due to the activity of CFS of the probiont. The reduction in bacterial population happened in a dose-dependent manner, i.e. the greater the amount of CFS, the greater the reduction in population, for example, 1 and 2.5 log reduction of bacterial cells were observed upon addition of 1 and 5 ml CFS of day 1 culture respectively (Figure 7, box 2). Further, the reduction was also observed age-dependent, i.e. 1 ml volume of day 3 CFS was found sufficient to produce complete elimination of pathogenic bacterial cells, while the same effect could only be availed with 5 ml CFS from day 2 culture (Figure 6, compare boxes 4 with 3).</p>
<p><b>DISCUSSION</b><br>
  Emergence  of any disease and irrational use of the antibiotics by the hatchery  technicians without any explicit pathogenic infection evidence has become the  most common trend for decades in aquaculture sector. This indiscriminate use of  antibiotics, engendering multiple drug resistant bacteria, and failure to  identify new antibiotics make it a gigantic work to find any alternative  prophylactics. If antibiotics or disinfectants are used to kill bacteria, some  bacteria will survive. These will then grow rapidly once the antibiotics are  withdrawn and their competitors are removed. Any virulent pathogens that  re-enter the pond or hatchery tanks can then exchange genes with the resistant  bacteria and survive further doses of antibiotic treatment. Thus antibiotic  resistant strains of pathogens evolve rapidly.</p>
 <p style="text-align:center"><img src="img/book1_1_7.png"/> </p>
 <p><b><i>Figure 7: </i></b><i>Effect of  different volumes (1, 2 and 5 ml) of cell-free supernatants from different aged  (1, 2 and 3-day old) culture of Lactobacillus spp on growth of washed cells of pathogenic </i><i>AS<sub>1</sub>T<sub>5</sub> </i><i>isolate</i><b></b></p>
 <p>The  current study attempted to isolate pathogenic species from dead prawn larvae  and found at least four different bacteria, <i>Enterococcus</i> spp., <i>Enterobacter</i> spp, <i>Klebsiella</i> spp., and <i>Exiguobacterium</i> spp, as evidenced from  their molecular characterization are associated with prawn mortality. To  elucidate the MDR pattern of these pathogens, the isolated pathogens were  challenged with eighteen antibiotics to find out whether those are effective  against pathogens or not. From these it appears none of the drugs could be  prescribed exclusively that would be effective against all the tested  pathogens. This is a serious concern as far as the disease management is  concerned, hence requires alternative management of control of bacterial  disease. Evaluation of the efficacy of the drugs apart from the drugs tested in  this study, or increasing the strength of the tested drugs could be a further  direction. Introduction of indigenous probiotic technology could be a smart  approach to combat the disease threat<sub>(14-15)</sub> and such a strategy  requires empirical research. It may be mentioned that a good number of  probiotics of foreign origin were tested in some hatcheries and culture ponds  of Khulna  region and was not found to be effective.<sub>(28)</sub> This observation  therefore demands to discover new probiotics from the indigenous origin to be  effective in local environment.</p>
 <p>In  this study, the antagonistic activity of probiotic bacteria, <i>Lactobacillus</i> spp. to different pathogenic  isolates<sub>(19)</sub> was analyzed by two <i>in  vitro</i> experiments. The isolated <i>Lactobacillus</i> strains were largely found effective in growth inhibition of pathogenic  bacteria on a solid medium creating more than 15 mm zone of inhibition, as  revealed by a disc-diffusion assay (Figure 6). In a liquid culture, a fixed  population of pathogenic bacteria, isolate AS<sub>1</sub>T<sub>5</sub>,  later identified as <i>Enterococcus</i> spp decreased  in a dose-dependent manner following increasing addition of cell-free  supernatant (CFS) of <i>Lactobacillus</i> spp. that eventually eliminated the total bacterial count within 18 h of  incubation (Figure 7). Such a reduction was also attributed to the age-dependent  manner, because a mere 1 ml volume of CFS from day 3 culture of <i>Lactobacillus</i> was found sufficient to  produce complete elimination of count yielding at least 7 log cycle reduction  within 18 h of incubation. It therefore indicates that the reduction of  pathogenic bacterial growth was solely due to the inhibitory potential of CFS.  Taken together, it can be assumed to confer that the probiotic potential of  selected <i>Lactobacillus</i> spp. could be  attributed to the extracellular substances released from the <i>Lactobacillus </i>probionts, rather than the  whole cells. Although the inhibitory mechanism of the interaction was not  characterized in this study, however, previous works have identified that  probionts release antimicrobial compounds, e.g. antibiotics, bacteriocins,  lysozyme, proteases, and/or hydrogen peroxide, or organic acids that could  alter pH of the surrounding environment, resulting in growth inhibition.</p>
 <p><b>CONCLUSION</b><b> </b><br>
   In  essence, from this study we figured that the mass mortality in PL in mid-2012  in Khulna and Sathkhira regions of Bangladesh was  caused by bacterial infection, and the causative agents were identified as <i>Enterobacter</i> spp., <i>Klebsiella</i> <i>pneumoniae</i>, <i>Enterococcus</i> spp, <i>Exiguobacterium</i> spp. We also observed a significant amount of  multidrug resistant pathogenic bacteria from prawn aquaculture as none of the eighteen  tested chemotherapeutic drugs was found effective to be used as sole drug  against these pathogenic bacteria. Probiotic treatment offers a very promising  alternative to the use of antibiotics in fish and prawn aquaculture. This study  was able to demonstrate <i>Lactobacillus  spp. </i>as potent probiotic bacterial candidate effective in inhibiting the  proliferation of isolated prawn pathogens, which caused PL mortality in Khulna  and Sathkhira hatcheries in 2012.<sub>(19)</sub> Further study is needed to  elucidate the pathogenicity of the tested bacteria, the exact mode of action of  the observed beneficial effects of the probiotic bacteria and to understand the  possibilities and limitations of microbial control in aquaculture.</p>
 <p><b>ACKNOWLEDGEMENTS</b><br>
   The  authors thankfully acknowledge the logistic support and financial assistance  kindly provided by Department of Fisheries, Ministry of Fisheries and  Livestock; and Ministry of Science and Technology, Government of the People&rsquo;s  Republic of Bangladesh  respectively in conducting this study.</p>
 <p><b>REFERENCES</b></p>
 <ol>
   <li>Azad  SA. Fisheries sector of Bangladesh  achievements and potentials. National Fish Week. 2013. 17-20.</li>
   <li>Lundin  GG. Fish health and quarantine. In Global Attempts to Address Shrimp Disease. Marine  Environmental Paper No. 4. 2006. 45. </li>
   <li>Boaventura  M, Canuto A, Ferreira A. Novas diretrizes no cultivo de camar˜ao cinza  Litopenaeus vannamei para o controle das enfermidades, Revista Aquicultura  &amp; Pesca. vol. 17. 2006. 25–28.</li>
   <li>Alavandi  SV, Vijayan KK, Santiago TC, Poornima M, Jithendran KP, Ali SA, Rajan JJS. Evaluation  of&nbsp;<i>Pseudomonas</i>&nbsp;sp. PM 11  and&nbsp;<i>Vibrio fluvialis</i>&nbsp;PM 17  on immune indices of tiger shrimp,&nbsp;<i>Penaeus  monodon</i>. Fish Shellfish Immunol. 2004. 17. 115-120.</li>
   <li>Jayaprakash NS, Kumar VJR, Philip R, Singh ISB. Vibrios  associated with&nbsp;<i>Macrobrachium  rosenbergii</i>&nbsp;(De Man, 1879) Larvae from three hatcheries on the Indian  southwest coast. Aquacult. Res. 2006. 37, 351-358.</li>
   <li>Kennedy  B, Venugopal MN, Karunasagar I.&nbsp;Bacterial flora  associated with the giant freshwater prawn <i>Macrobrachium  rosenbergii</i>&nbsp;in the hatchery system. Aquaculture. 2006; 261, 1156-1167.</li>
   <li>Lightner  DV. Diseases of Cultured Penaeid Shrimp. In: Handbook of Marine Culture, Vol.  1, Crustacean Aquaculture. McVey, J.P. CRC Press, Boca Raton, FL. USA., ISBN: 0849302552. 1993.  393-486.</li>
   <li>Ruangpan  L, Danayadol Y, Direkbusarakom S, Siurairastana S, Flegel TW. Lethal toxicity  of&nbsp;Vibrio harveyi&nbsp;to cultivated <i>Penaues  mondon</i>&nbsp;induced by a bacteriophage. Dis. Aquat. Org. 1999. 35, 195-201.</li>
   <li>Vandenberghe  J, Verdonch L, Robles-Arozarena R, Rivera G, Bolland A,&nbsp;et al.&nbsp;Vibrios  associated with&nbsp;<i>Litopenaeus vannamei</i> larvae, postlarvae, broodstock and hatchery probionts. Applied Environ.  Microbiol. 1999. 65, 2592-2597.</li>
   <li>Vaseeharan B, Ramasamy D.&nbsp;Control  of pathogenic&nbsp;<i>Vibrio</i>&nbsp;spp. by&nbsp;<i>Bacillus</i> <i>subtilis</i>&nbsp;BT 23, a possible probiotic treatment for black tiger  shrimp&nbsp;<i>Penaeus mondon</i>. Lett.  Applied Microbiol. 2003; 36, 83-87.</li>
   <li>Chiu C, Guu Y, Liu C, Pan T, Cheng W.  Immune responses and gene expression in white shrimp, <i>Litopenaeus vannamei</i>, induced by <i>Lactobacillus plantarum</i>. Fish Shellfish Immunol. 2007. 23, 364–377.</li>
   <li>Westerdahi A, Olsson JC, Kjelleberg  S, Conway PL. Isolation and characterization of turbot (<i>Scophtaimus maximus</i>) associated bacteria with inhibitory effects  against <i>Vibrio anguillarum</i>. Appl. Environ.  Microbiol. 1991. 57(8): 2223-8.</li>
   <li>Verschuere L, Rombaut G, Sorgeloos P,  Verstraere W. Probiotic bacteria as biological control agent in aquaculture.  Microbial Mol. Biol. Rev. 2000. (4): 655-671.</li>
   <li>Rahman S, Khan SN, Naser MN,  Karim MM. Application of probiotic bacteria: A novel approach towards ensuring food  safety in shrimp aquaculture.Journal of Bangladesh Academy of Sciences. 2009. 33(1):  139-144.</li>
   <li>Rahman S, Khan SN, Naser  MN, Karim MM. Safety issues of isolated probiotic  natured bacteria from Bangladesh  coastal waters for controlling shrimp diseases. J. Sci. Res. 2011. 3(3): 659-66. </li>
   <li>Moriarty DJW. Disease control in  shrimp aquaculture with probitic bacteria. In: Microbial biosystems; new  frontier – proceedings in 8th intl. Symp on Microbial Ecology eds.  CR Bell et al (Atlantic Canada Society for Microbial Ecology. Halifax. Canada.1999).</li>
   <li>Reuter G. Probiotic-possibilities and  limitation of their application in food, animal feed, and in pharmaceutical  preparation for men and animals. Berl Munch Tieraztl Wochenschr. 2001. 144(11-12):  410-9.</li>
   <li>El-Ghaisha  S, Ahmadovaa A, Hadji-Sfaxia I, El Mecherfia KE, Bazukyane I, Choiseta Y,  Rabesonaa H, Sitohya M, Popove YG, Kuliev AA, Mozzig F, Choberta JM, Haertle T.  Potential use of lactic acid bacteria for reduction of allergenicity and for  longer conservation of fermented foods. Trends Food Sci Tech. 2011. 22: 509-516.</li>
   <li>Ramim  AM, Rahman S, Ahsan DA, Naser MN,  Karim MM. Addressing mortality of post-larvae in Golda hatcheries of Sathkhira  and Khulna region of Bangladesh. In:  Proceedings of the 9th Annual  Scientific Conference, Published by Chittagong  Veterinary and Animal   Sciences University.  2013. 137-45.</li>
   <li>Bauer  AW, Kirby WM, Sherris JC, Turck M. Antibiotic susceptibility testing by a  standardized single disk method. Am J Clin Pathol. 1966. 45: 493–496. </li>
   <li>Lane DJ.  16S/23S rRNA sequencing, in Stackebrandt E, Goodfellow M (Ed.). Nucleic acid  techniques in bacterial systematic, England. John Wiley &amp; Sons, Chichester. 1991; 115-163.</li>
   <li>Frank JA,  Reich CI, Sharma S, Weisbaum JS, Wilson  BA, Olsen GJ. Critical Evaluation of Two Primers Commonly Used for  Amplification of Bacterial 16S rRNA Genes. Applied and Environmental  Microbiology. 2008. 74(8): 2461–2470.</li>
   <li>Vaneechoutte  M, Dijkshoorn L, Tjernberg I, Elaichouni A, de Vos P, Claeys G, Verschraegen G.  Identification of Acinetobacter genomic species by amplified ribosomal DNA  restriction analysis. J Clin Microbiol. 1995. 33: 11–15.</li>
   <li>Tamura  K, Dudley J, Nei M, Kumar S. Molecular Evolutionary Genetics Analysis (MEGA)  software version 4.0. Mol. Biol. Evol. 2007. 24:1596–1599.</li>
   <li>Hoque  FA, Hossain KM, Rahman MSM, Billah MM, Islam KMD. &quot;Isolation, identification  and analysis of probiotic properties of <i>Lactobacillus</i> spp. from selective regional yoghurts.&quot; World Journal of Dairy &amp; Food  Sciences. 2010. 5(1): 39-46. </li>
   <li>Sarker A, Khan SN, Naser MN,  Karim MM. Isolation of probiotic bacteria from natural sources to control diseases  in shrimp aquaculture. Food Safety &amp; Hygiene. Bangladesh Academy  of Sciences. ISBN: 984-300-001933-4. 2008; 129-135. </li>
   <li>Chythanya R, Karunasagar I,  Karunasagar I. Inhibition of shrimp pathogenic vibrios by a marine <i>Pseudomonas</i> I-2 strain. Aquaculture.  2002. 208: 1-10. </li>
   <li>Alam MJ. CSO, Bangladesh  Fisheries Research Institute - Personal communication. 2009. </li>
 </ol>
        </div>
<!-- .News -->
		
		
	
	
    

        </div></div>
        <!-- .Content -->
    
    </div>
    <!-- .MAIN -->

</div>
<!-- .CONTENT -->



<!-- BOTTOM LINE -->
<div id="bottomLine" class="width100">
	<div class="width1000">
        <div id="bottomNav">
		<ul id="menu-footer" class="">
        <!--<li id="menu-item-1092" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-60 current_page_item menu-item-1092"><a href="index.html">Home</a></li>
<li id="menu-item-1093" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1093"><a href="#">CARES</a></li>
<li id="menu-item-1100" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1100"><a href="#">Function & Administration</a></li>
<li id="menu-item-1128" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1128"><a href="#">Projects</a></li>
<li id="menu-item-1095" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1095"><a href="#">News</a></li>
<li id="menu-item-1094" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1094"><a href="#">Contact us</a></li>-->
</ul>        </div>
        
        <div id="credits"><a href="http://www.webpers.com/" title="Webpers"><img src="wp-content/uploads/sites/5/2013/01/footer_webpers.png" alt="Image 1"></a> </div>
	</div>
</div>
<!-- .BOTTOM LINE -->


<script type='text/javascript' src='../wp-content/plugins/woocommerce/assets/js/frontend/add-to-cart.min7514.js?ver=1.6.6'></script>
<script type='text/javascript' src='../wp-content/plugins/woocommerce/assets/js/jquery-plugins.min7514.js?ver=1.6.6'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var woocommerce_params = {"countries":"{\"AU\":{\"ACT\":\"Australian Capital Territory\",\"NSW\":\"New South Wales\",\"NT\":\"Northern Territory\",\"QLD\":\"Queensland\",\"SA\":\"South Australia\",\"TAS\":\"Tasmania\",\"VIC\":\"Victoria\",\"WA\":\"Western Australia\"},\"AF\":[],\"AT\":[],\"BE\":[],\"BI\":[],\"BR\":{\"AM\":\"Amazonas\",\"AC\":\"Acre\",\"AL\":\"Alagoas\",\"AP\":\"Amap\u00e1\",\"CE\":\"Cear\u00e1\",\"DF\":\"Distrito Federal\",\"ES\":\"Esp\u00edrito Santo\",\"MA\":\"Maranh\u00e3o\",\"PR\":\"Paran\u00e1\",\"PE\":\"Pernambuco\",\"PI\":\"Piau\u00ed\",\"RN\":\"Rio Grande do Norte\",\"RS\":\"Rio Grande do Sul\",\"RO\":\"Rond\u00f4nia\",\"RR\":\"Roraima\",\"SC\":\"Santa Catarina\",\"SE\":\"Sergipe\",\"TO\":\"Tocantins\",\"PA\":\"Par\u00e1\",\"BH\":\"Bahia\",\"GO\":\"Goi\u00e1s\",\"MT\":\"Mato Grosso\",\"MS\":\"Mato Grosso do Sul\",\"RJ\":\"Rio de Janeiro\",\"SP\":\"S\u00e3o Paulo\",\"MG\":\"Minas Gerais\",\"PB\":\"Para\u00edba\"},\"CA\":{\"AB\":\"Alberta\",\"BC\":\"British Columbia\",\"MB\":\"Manitoba\",\"NB\":\"New Brunswick\",\"NF\":\"Newfoundland\",\"NT\":\"Northwest Territories\",\"NS\":\"Nova Scotia\",\"NU\":\"Nunavut\",\"ON\":\"Ontario\",\"PE\":\"Prince Edward Island\",\"QC\":\"Quebec\",\"SK\":\"Saskatchewan\",\"YT\":\"Yukon Territory\"},\"CZ\":[],\"DE\":[],\"DK\":[],\"FI\":[],\"FR\":[],\"HK\":{\"HONG KONG\":\"Hong Kong Island\",\"KOWLOON\":\"Kowloon\",\"NEW TERRITORIES\":\"New Territories\"},\"HU\":[],\"IS\":[],\"IL\":[],\"NL\":{\"DR\":\"Drenthe\",\"FL\":\"Flevoland\",\"FR\":\"Friesland\",\"GLD\":\"Gelderland\",\"GRN\":\"Groningen\",\"LB\":\"Limburg\",\"NB\":\"Noord-Brabant\",\"NH\":\"Noord-Holland\",\"OV\":\"Overijssel\",\"UT\":\"Utrecht\",\"ZLD\":\"Zeeland\",\"ZH\":\"Zuid-Holland\"},\"NZ\":{\"NL\":\"Northland\",\"AK\":\"Auckland\",\"WA\":\"Waikato\",\"BP\":\"Bay of Plenty\",\"TK\":\"Taranaki\",\"HB\":\"Hawke\u2019s Bay\",\"MW\":\"Manawatu-Wanganui\",\"WE\":\"Wellington\",\"NS\":\"Nelson\",\"MB\":\"Marlborough\",\"TM\":\"Tasman\",\"WC\":\"West Coast\",\"CT\":\"Canterbury\",\"OT\":\"Otago\",\"SL\":\"Southland\"},\"NO\":[],\"PL\":[],\"SG\":[],\"SK\":[],\"SI\":[],\"LK\":[],\"SE\":[],\"US\":{\"AL\":\"Alabama\",\"AK\":\"Alaska\",\"AZ\":\"Arizona\",\"AR\":\"Arkansas\",\"CA\":\"California\",\"CO\":\"Colorado\",\"CT\":\"Connecticut\",\"DE\":\"Delaware\",\"DC\":\"District Of Columbia\",\"FL\":\"Florida\",\"GA\":\"Georgia\",\"HI\":\"Hawaii\",\"ID\":\"Idaho\",\"IL\":\"Illinois\",\"IN\":\"Indiana\",\"IA\":\"Iowa\",\"KS\":\"Kansas\",\"KY\":\"Kentucky\",\"LA\":\"Louisiana\",\"ME\":\"Maine\",\"MD\":\"Maryland\",\"MA\":\"Massachusetts\",\"MI\":\"Michigan\",\"MN\":\"Minnesota\",\"MS\":\"Mississippi\",\"MO\":\"Missouri\",\"MT\":\"Montana\",\"NE\":\"Nebraska\",\"NV\":\"Nevada\",\"NH\":\"New Hampshire\",\"NJ\":\"New Jersey\",\"NM\":\"New Mexico\",\"NY\":\"New York\",\"NC\":\"North Carolina\",\"ND\":\"North Dakota\",\"OH\":\"Ohio\",\"OK\":\"Oklahoma\",\"OR\":\"Oregon\",\"PA\":\"Pennsylvania\",\"RI\":\"Rhode Island\",\"SC\":\"South Carolina\",\"SD\":\"South Dakota\",\"TN\":\"Tennessee\",\"TX\":\"Texas\",\"UT\":\"Utah\",\"VT\":\"Vermont\",\"VA\":\"Virginia\",\"WA\":\"Washington\",\"WV\":\"West Virginia\",\"WI\":\"Wisconsin\",\"WY\":\"Wyoming\"},\"USAF\":{\"AA\":\"Americas\",\"AE\":\"Europe\",\"AP\":\"Pacific\"},\"VN\":[]}","select_state_text":"Select an option\u2026","plugin_url":"http:\/\/themeblossom.net\/thecause\/wp-content\/plugins\/woocommerce","ajax_url":"\/\/themeblossom.net\/thecause\/wp-admin\/admin-ajax.php","ajax_loader_url":"http:\/\/themeblossom.net\/thecause\/wp-content\/plugins\/woocommerce\/assets\/images\/ajax-loader.gif","required_rating_text":"Please select a rating","review_rating_required":"yes","required_text":"required","update_order_review_nonce":"8b91ed7dbe","apply_coupon_nonce":"d2255403b8","option_guest_checkout":"yes","checkout_url":"\/\/themeblossom.net\/thecause\/wp-admin\/admin-ajax.php?action=woocommerce-checkout","is_checkout":"0","update_shipping_method_nonce":"37a40eae60","add_to_cart_nonce":"80a48b5bd7"};
/* ]]> */
</script>
<script type='text/javascript' src='../wp-content/plugins/woocommerce/assets/js/frontend/woocommerce.min7514.js?ver=1.6.6'></script>

</body>

<!-- Mirrored from themeblossom.net/thecause/blog/ by HTTrack Website Copier/3.x [XR&CO'2010], Wed, 13 Mar 2013 20:13:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8" /><!-- /Added by HTTrack -->
</html>